### ISSUE TITLE
The offiline mode doesn't works, could you fix this?

### ISSUE THREAD

**Issue by nazarepiedady** [freeCodeCamp/devdocs#770]

**Comment by nazare3000** [comment]
I use the firefox web browser developer edition, and I'm so matters with the offiline mode of devdocs tools

**Comment by j-f1** [comment]
Do you get any error messages if you go to https://devdocs.io/offline?

**Comment by Thibaut** [comment]
If the error you're seeing is "UnknownError - The operation failed for reasons unrelated to the database itself and not covered by any other error code", this is caused by a [Firefox bug](https://bugzilla.mozilla.org/show_bug.cgi?id=944918). To fix, go to `about:support`, open your profile folder, delete the `storage` folder, and restart Firefox.

**Comment by IanChen83** [comment]
I'm using Firefox 59.0.2 (64bit), and this happens only when the computer is not connected to the Internet. Here is the error message from the developer tools console.

````
SyntaxError: expected expression, got '<'  application-a76771ca96b817386225c34c635f7e7a443a946cc92dc4b820f3099252860505.js:1
ReferenceError: app is not defined   
docs-4bacf16ef051326fd44767654e3c4f5e1c1aacc4e0e08fae67fe77d9fcb270a2.js:1:1
````

I also open `/assets/application-a76771ca96b817386225c34c635f7e7a443a946cc92dc4b820f3099252860505.js` and find out it is an HTML file.

**Comment by ngomile** [comment]
I have the same problem as @IanChen83. I am using Firefox 60.0.

![devdocs issue](https://user-images.githubusercontent.com/39145128/39850205-63e88670-5410-11e8-9966-fc494dbb1e90.png)

**Comment by E314c** [comment]
I'm seeing similar to the two above:
Looks like the `application-6b39fc8926f0f5a2bece9c2e2b1af79bd06190e9bc1d802ea660eed99ffdafed.js` file it's trying to load is actually an HTML file:
![seemstoattemptloadinghtmlasjavascript](https://user-images.githubusercontent.com/6907531/41037024-e8da6dd8-6989-11e8-98f1-89fbeda0f0c1.png)
I can confirm that the offline data is present in my local storage:
![datastillexists](https://user-images.githubusercontent.com/6907531/41037051-fbb7b4d8-6989-11e8-9d50-4e457cc18762.png)

Perhaps worth noting I've have devdocs working in offline mode on this machine for ages and it has only recently begun to fail. Currently running Firefox Dev 61.0b11, but have always been running FF.

As an aside, I booted up chrome on my machine and, for the first time, navigated to devdocs.io and setup offline data and proved it will work in offline mode.

This suggests it's either an issue with FF, or somehow the cache of the main webpage has been corrupted on my FF instance.

How exactly is the initial webpage stored in the browser to be loaded in offline mode?

**Comment by E314c** [comment]
I have tried `Ctrl+F5` to no avail.
Also tried looking for a serviceWorker in FF's `about:debugging#workers`, but couldn't see anything.

**Comment by E314c** [comment]
To add to this: I manually went through my cache, local storage and cookies and wiped all `devdocs.io` related data and it then began to work again: I reinstalled a few documents for offline and it was functional.

A little later I then added _quite a few more_ documents to offline mode (~100mb of docs), at which point the problem occured again, with the offline page trying to load an HTML file as javascript.

Again, my local storage is still there and intact (from what I can see). 
I've just found that FF lists app cache max size and used under `about:cache`, so i'll check that next time i'm on that machine and feedback.

Though I reckon this'll all be fixed by moving over to service workers: #774
